<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="XamlParseException.xml" source-language="en-US" target-language="de-DE">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac5f7785f59cbc386e6af15aa0c6eb7b76532178de9.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">f7785f59cbc386e6af15aa0c6eb7b76532178de9</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.Windows.Markup.XamlParseException">
          <source>Represents the exception class for parser-specific exceptions from a WPF XAML parser.</source>
          <target state="translated">Stellt die Ausnahmeklasse für parserspezifische Ausnahmen eines WPF-XAML-Parsers dar.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" uid="T:System.Windows.Markup.XamlParseException">
          <source>This exception is used in XAML API or WPF XAML parser operations from .NET Framework 3.0 and .NET Framework 3.5, or for specific use of the WPF XAML parser by calling <ph id="ph1">&lt;see cref="T:System.Windows.Markup.XamlReader" /&gt;</ph> API.</source>
          <target state="translated">Diese Ausnahme wird in XAML-API- oder WPF-XAML-Parservorgängen in .NET Framework 3.0 und .NET Framework 3.5 verwendet, oder sie ist für die spezifische Verwendung des WPF-XAML-Parsers durch Aufrufen der <ph id="ph1">&lt;see cref="T:System.Windows.Markup.XamlReader" /&gt;</ph>-API bestimmt.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Markup.XamlParseException">
          <source><ph id="ph1">&lt;xref:System.Windows.Markup.XamlParseException&gt;</ph> is used only for the WPF-implemented XAML parser that performs the XAML parsing and loading for WPF applications.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Windows.Markup.XamlParseException&gt;</ph> wird nur für WPF implementierten XAML-Parser verwendet, analysieren und Laden von XAML für WPF-Anwendungen ausführt.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Markup.XamlParseException">
          <source>Specifically, the exception is only relevant when an application targets <ph id="ph1">[!INCLUDE[net_v30_short](~/includes/net-v30-short-md.md)]</ph> and <ph id="ph2">[!INCLUDE[net_v35_short](~/includes/net-v35-short-md.md)]</ph>.</source>
          <target state="translated">Insbesondere ist die Ausnahme nur relevant, wenn eine Anwendung Ziele <ph id="ph1">[!INCLUDE[net_v30_short](~/includes/net-v30-short-md.md)]</ph> und <ph id="ph2">[!INCLUDE[net_v35_short](~/includes/net-v35-short-md.md)]</ph>.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Markup.XamlParseException">
          <source>The exception can also originate from user code in run-time calls to APIs that hook up the WPF-implemented XAML parser to load XAML from within a running WPF application (for example, calls to <ph id="ph1">&lt;xref:System.Windows.Markup.XamlReader.Load%2A?displayProperty=nameWithType&gt;</ph>).</source>
          <target state="translated">Die Ausnahme kann auch aus Benutzercode in zur Laufzeit Aufrufen von APIs, die die WPF implementierten XAML-Parser zum Laden von XAML innerhalb einer ausgeführten WPF-Anwendung einbinden stammen (z. B. Aufrufe von <ph id="ph1">&lt;xref:System.Windows.Markup.XamlReader.Load%2A?displayProperty=nameWithType&gt;</ph>).</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Markup.XamlParseException">
          <source>For <ph id="ph1">[!INCLUDE[net_v40_short](~/includes/net-v40-short-md.md)]</ph>, the <ph id="ph2">&lt;xref:System.Xaml.XamlParseException&gt;</ph> exception that typically reports XAML processing exceptions is defined in a different namespace (<ph id="ph3">&lt;xref:System.Xaml&gt;</ph>) and a different assembly (System.Xaml).</source>
          <target state="translated">Für <ph id="ph1">[!INCLUDE[net_v40_short](~/includes/net-v40-short-md.md)]</ph>, <ph id="ph2">&lt;xref:System.Xaml.XamlParseException&gt;</ph> Ausnahme, die Ausnahmen bei der Verwendung von XAML-Verarbeitung in der Regel Berichte in einem anderen Namespace definiert ist (<ph id="ph3">&lt;xref:System.Xaml&gt;</ph>) und einer anderen Assembly ("System.xaml").</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Markup.XamlParseException">
          <source>Unless you are writing an equivalent to the WPF XAML parser or working with <ph id="ph1">[!INCLUDE[net_v30_short](~/includes/net-v30-short-md.md)]</ph> and <ph id="ph2">[!INCLUDE[net_v35_short](~/includes/net-v35-short-md.md)]</ph> targeting, you generally will not throw <ph id="ph3">&lt;xref:System.Windows.Markup.XamlParseException&gt;</ph> from your own code.</source>
          <target state="translated">Es sei denn, Sie schreiben eine Entsprechung für den WPF XAML-Parser oder Arbeiten mit <ph id="ph1">[!INCLUDE[net_v30_short](~/includes/net-v30-short-md.md)]</ph> und <ph id="ph2">[!INCLUDE[net_v35_short](~/includes/net-v35-short-md.md)]</ph> verwenden möchten, Sie in der Regel keine Ausnahme auslöst, <ph id="ph3">&lt;xref:System.Windows.Markup.XamlParseException&gt;</ph> aus Ihrem eigenen Code.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Markup.XamlParseException">
          <source>However, handling for the exception is sometimes necessary.</source>
          <target state="translated">Behandlung der Ausnahme ist jedoch manchmal erforderlich.</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Markup.XamlParseException">
          <source>For application scenarios, where you may want to suppress XAML parse errors, a <ph id="ph1">&lt;xref:System.Windows.Threading.Dispatcher&gt;</ph> <ph id="ph2">&lt;xref:System.Windows.Threading.Dispatcher.UnhandledException&gt;</ph> event handler at the application level is one way to handle a run-time <ph id="ph3">&lt;xref:System.Windows.Markup.XamlParseException&gt;</ph>.</source>
          <target state="translated">Für die Anwendungsszenarien, in denen Sie möglicherweise XAML-Analysefehler unterdrücken möchten, eine <ph id="ph1">&lt;xref:System.Windows.Threading.Dispatcher&gt;</ph> <ph id="ph2">&lt;xref:System.Windows.Threading.Dispatcher.UnhandledException&gt;</ph> -Ereignishandler auf Anwendungsebene ist eine Möglichkeit, eine Laufzeit-behandeln <ph id="ph3">&lt;xref:System.Windows.Markup.XamlParseException&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Markup.XamlParseException">
          <source>Whether to suppress exceptions or let them surface to user code depends on how you design your application for purposes of loading XAML, and the trust level that you assign to the XAML your application loads.</source>
          <target state="translated">Hängt von, ob Ausnahmen unterdrückt, oder informieren sie sich in Benutzercode wie Ihrer Anwendung zum Zweck Entwerfen der Laden von XAML und lädt die Vertrauensebene der XAML-Code die Anwendung zuweisen.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Markup.XamlParseException">
          <source>For more information, see <bpt id="p1">[</bpt>XAML Security Considerations<ept id="p1">](~/docs/framework/xaml-services/xaml-security-considerations.md)</ept> or "XAML Security" section of <bpt id="p2">[</bpt>XAML Overview (WPF)<ept id="p2">](~/docs/framework/wpf/advanced/xaml-overview-wpf.md)</ept>.</source>
          <target state="translated">Weitere Informationen finden Sie unter <bpt id="p1">[</bpt>XAML-Sicherheitsüberlegungen<ept id="p1">](~/docs/framework/xaml-services/xaml-security-considerations.md)</ept> oder im Abschnitt "XAML-Sicherheit" <bpt id="p2">[</bpt>XAML Overview (WPF)<ept id="p2">](~/docs/framework/wpf/advanced/xaml-overview-wpf.md)</ept>.</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Markup.XamlParseException">
          <source>For pages of an application, when the <ph id="ph1">&lt;xref:System.Windows.Markup.XamlParseException&gt;</ph> is thrown, it is usually in the context of the <ph id="ph2">`InitializeComponent`</ph> call made by your page class, which is the entry point for the WPF application model's usage of the WPF XAML parser at the per-page level.</source>
          <target state="translated">Für Seiten einer Anwendung Wenn die <ph id="ph1">&lt;xref:System.Windows.Markup.XamlParseException&gt;</ph> ist ausgelöst wird, ist dies normalerweise im Rahmen der <ph id="ph2">`InitializeComponent`</ph> Aufruf durch die Page-Klasse, der Einstiegspunkt für das WPF-Anwendungsmodell Verwendung der WPF XAML-Parser Ebene der pro Seite ist.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Markup.XamlParseException">
          <source>Therefore another possible handling strategy is to place <ph id="ph1">`try/catch`</ph> blocks in <ph id="ph2">`InitializeComponent`</ph>.</source>
          <target state="translated">Aus diesem Grund eine weitere mögliche Behandlung von Strategie besteht darin zu platzieren <ph id="ph1">`try/catch`</ph> Blöcke in <ph id="ph2">`InitializeComponent`</ph>.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Markup.XamlParseException">
          <source>However, this technique does not integrate well with templates, visual design surfaces and other generated sources that hook up <ph id="ph1">`InitializeComponent`</ph>.</source>
          <target state="translated">Allerdings diese Technik lässt sich nicht gut mit Vorlagen, visuelle Entwurfsoberflächen integrieren und andere generierte Quellen verknüpft sich <ph id="ph1">`InitializeComponent`</ph>.</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" uid="T:System.Windows.Markup.XamlParseException">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Windows.Markup.XamlParseException" /&gt;</ph> class.</source>
          <target state="translated">Initialisiert eine neue Instanz der <ph id="ph1">&lt;see cref="T:System.Windows.Markup.XamlParseException" /&gt;</ph>-Klasse.</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" uid="M:System.Windows.Markup.XamlParseException.#ctor">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Windows.Markup.XamlParseException" /&gt;</ph> class.</source>
          <target state="translated">Initialisiert eine neue Instanz der <ph id="ph1">&lt;see cref="T:System.Windows.Markup.XamlParseException" /&gt;</ph>-Klasse.</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" uid="M:System.Windows.Markup.XamlParseException.#ctor(System.String)">
          <source>The exception message.</source>
          <target state="translated">Die Ausnahmemeldung.</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" uid="M:System.Windows.Markup.XamlParseException.#ctor(System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Windows.Markup.XamlParseException" /&gt;</ph> class, using the specified exception message string.</source>
          <target state="translated">Initialisiert eine neue Instanz der <ph id="ph1">&lt;see cref="T:System.Windows.Markup.XamlParseException" /&gt;</ph>-Klasse mit der angegebenen Ausnahmemeldungszeichenfolge.</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" uid="M:System.Windows.Markup.XamlParseException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Contains all the information that is required to serialize or deserialize the object.</source>
          <target state="translated">Enthält alle zum Serialisieren oder Deserialisieren des Objekts erforderlichen Informationen.</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" uid="M:System.Windows.Markup.XamlParseException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The source and destination of a serialized stream.</source>
          <target state="translated">Die Quelle und das Ziel eines serialisierten Streams.</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" uid="M:System.Windows.Markup.XamlParseException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Windows.Markup.XamlParseException" /&gt;</ph> class.</source>
          <target state="translated">Initialisiert eine neue Instanz der <ph id="ph1">&lt;see cref="T:System.Windows.Markup.XamlParseException" /&gt;</ph>-Klasse.</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Windows.Markup.XamlParseException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>This constructor is used for serialization when marshaling an exception of this type across an application domain or machine boundary.</source>
          <target state="translated">Dieser Konstruktor wird für die Serialisierung verwendet, wenn eine Ausnahme dieses Typs über eine Anwendung Domänen- oder -Grenze hinweg zu marshallen.</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve" uid="M:System.Windows.Markup.XamlParseException.#ctor(System.String,System.Exception)">
          <source>The exception message.</source>
          <target state="translated">Die Ausnahmemeldung.</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve" uid="M:System.Windows.Markup.XamlParseException.#ctor(System.String,System.Exception)">
          <source>The initial exception that occurred.</source>
          <target state="translated">Die anfänglich aufgetretene Ausnahme.</target>       </trans-unit>
        <trans-unit id="125" translate="yes" xml:space="preserve" uid="M:System.Windows.Markup.XamlParseException.#ctor(System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Windows.Markup.XamlParseException" /&gt;</ph> class, using the specified exception message string and inner exception.</source>
          <target state="translated">Initialisiert eine neue Instanz der <ph id="ph1">&lt;see cref="T:System.Windows.Markup.XamlParseException" /&gt;</ph>-Klasse mit der angegebenen Meldungszeichenfolge und der angegebenen internen Ausnahme.</target>       </trans-unit>
        <trans-unit id="126" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Windows.Markup.XamlParseException.#ctor(System.String,System.Exception)">
          <source>An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception.</source>
          <target state="translated">Eine Ausnahme, die als direktes Ergebnis einer vorhergehenden Ausnahme ausgelöst wird sollte einen Verweis auf die vorhergehende Ausnahme verfügen.</target>       </trans-unit>
        <trans-unit id="127" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Windows.Markup.XamlParseException.#ctor(System.String,System.Exception)">
          <source>The <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property returns the same value that is passed into the constructor, or <ph id="ph2">`null`</ph> if the <ph id="ph3">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property does not supply the inner exception value to the constructor.</source>
          <target state="translated">Die <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>-Eigenschaft gibt den gleichen Wert zurück, der an den Konstruktor übergeben wird, oder <ph id="ph2">`null`</ph>, wenn die <ph id="ph3">&lt;xref:System.Exception.InnerException%2A&gt;</ph>-Eigenschaft den Wert der inneren Ausnahme nicht an den Konstruktor übergibt.</target>       </trans-unit>
        <trans-unit id="128" translate="yes" xml:space="preserve" uid="M:System.Windows.Markup.XamlParseException.#ctor(System.String,System.Int32,System.Int32)">
          <source>The exception message.</source>
          <target state="translated">Die Ausnahmemeldung.</target>       </trans-unit>
        <trans-unit id="129" translate="yes" xml:space="preserve" uid="M:System.Windows.Markup.XamlParseException.#ctor(System.String,System.Int32,System.Int32)">
          <source>The line number where the exception occurred.</source>
          <target state="translated">Die Nummer der Zeile, in der die Ausnahme aufgetreten ist.</target>       </trans-unit>
        <trans-unit id="130" translate="yes" xml:space="preserve" uid="M:System.Windows.Markup.XamlParseException.#ctor(System.String,System.Int32,System.Int32)">
          <source>The position in the line at which the exception occurred.</source>
          <target state="translated">Die Position in der Zeile, bei der die Ausnahme aufgetreten ist.</target>       </trans-unit>
        <trans-unit id="131" translate="yes" xml:space="preserve" uid="M:System.Windows.Markup.XamlParseException.#ctor(System.String,System.Int32,System.Int32)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Windows.Markup.XamlParseException" /&gt;</ph> class, using the specified exception message string, and the specified line number and position in the line.</source>
          <target state="translated">Initialisiert eine neue Instanz der <ph id="ph1">&lt;see cref="T:System.Windows.Markup.XamlParseException" /&gt;</ph>-Klasse mit der angegebenen Ausnahmemeldungszeichenfolge und der angegebenen Zeilennummer und Position in der Zeile.</target>       </trans-unit>
        <trans-unit id="132" translate="yes" xml:space="preserve" uid="M:System.Windows.Markup.XamlParseException.#ctor(System.String,System.Int32,System.Int32,System.Exception)">
          <source>The exception message.</source>
          <target state="translated">Die Ausnahmemeldung.</target>       </trans-unit>
        <trans-unit id="133" translate="yes" xml:space="preserve" uid="M:System.Windows.Markup.XamlParseException.#ctor(System.String,System.Int32,System.Int32,System.Exception)">
          <source>The line number where the exception occurred.</source>
          <target state="translated">Die Nummer der Zeile, in der die Ausnahme aufgetreten ist.</target>       </trans-unit>
        <trans-unit id="134" translate="yes" xml:space="preserve" uid="M:System.Windows.Markup.XamlParseException.#ctor(System.String,System.Int32,System.Int32,System.Exception)">
          <source>The position in the line at which the exception occurred.</source>
          <target state="translated">Die Position in der Zeile, bei der die Ausnahme aufgetreten ist.</target>       </trans-unit>
        <trans-unit id="135" translate="yes" xml:space="preserve" uid="M:System.Windows.Markup.XamlParseException.#ctor(System.String,System.Int32,System.Int32,System.Exception)">
          <source>The initial exception that occurred.</source>
          <target state="translated">Die anfänglich aufgetretene Ausnahme.</target>       </trans-unit>
        <trans-unit id="136" translate="yes" xml:space="preserve" uid="M:System.Windows.Markup.XamlParseException.#ctor(System.String,System.Int32,System.Int32,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Windows.Markup.XamlParseException" /&gt;</ph> class, using the specified exception message, inner exception, line number, and position in the line.</source>
          <target state="translated">Initialisiert eine neue Instanz der <ph id="ph1">&lt;see cref="T:System.Windows.Markup.XamlParseException" /&gt;</ph>-Klasse mit der angegebenen Ausnahmemeldung, inneren Ausnahme, Zeilennummer und Position in der Zeile.</target>       </trans-unit>
        <trans-unit id="137" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Windows.Markup.XamlParseException.#ctor(System.String,System.Int32,System.Int32,System.Exception)">
          <source>An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception.</source>
          <target state="translated">Eine Ausnahme, die als direktes Ergebnis einer vorhergehenden Ausnahme ausgelöst wird sollte einen Verweis auf die vorhergehende Ausnahme verfügen.</target>       </trans-unit>
        <trans-unit id="138" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Windows.Markup.XamlParseException.#ctor(System.String,System.Int32,System.Int32,System.Exception)">
          <source>The <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property returns the same value that is passed into the constructor, or <ph id="ph2">`null`</ph> if the <ph id="ph3">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property does not supply the inner exception value to the constructor.</source>
          <target state="translated">Die <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>-Eigenschaft gibt den gleichen Wert zurück, der an den Konstruktor übergeben wird, oder <ph id="ph2">`null`</ph>, wenn die <ph id="ph3">&lt;xref:System.Exception.InnerException%2A&gt;</ph>-Eigenschaft den Wert der inneren Ausnahme nicht an den Konstruktor übergibt.</target>       </trans-unit>
        <trans-unit id="139" translate="yes" xml:space="preserve" uid="P:System.Windows.Markup.XamlParseException.BaseUri">
          <source>Gets base URI information when the exception is thrown.</source>
          <target state="translated">Ruft Basis-URI-Informationen ab, wenn die Ausnahme ausgelöst wird.</target>       </trans-unit>
        <trans-unit id="140" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Markup.XamlParseException.BaseUri">
          <source>The parser context base URI.</source>
          <target state="translated">Der Basis-URI für den Parserkontext.</target>       </trans-unit>
        <trans-unit id="141" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Markup.XamlParseException.BaseUri">
          <source>Base URI is potentially available because of parser context when the exception occurred.</source>
          <target state="translated">Basis-URI ist aufgrund von Parserkontext potenziell verfügbar, wenn die Ausnahme aufgetreten ist.</target>       </trans-unit>
        <trans-unit id="142" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Markup.XamlParseException.BaseUri">
          <source>Knowing the base URI can be useful for debugging cases where a resource resolution error is the underlying cause.</source>
          <target state="translated">Zu wissen, der Basis kann URI nützlich zum Debuggen von Fällen, in denen eine Auflösung Ressourcenfehlers die zugrunde liegende Ursache.</target>       </trans-unit>
        <trans-unit id="143" translate="yes" xml:space="preserve" uid="M:System.Windows.Markup.XamlParseException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The serialization information object to add the serialization data to.</source>
          <target state="translated">Das Serialisierungsinfoobjekt, dem die Serialisierungsdaten hinzugefügt werden sollen.</target>       </trans-unit>
        <trans-unit id="144" translate="yes" xml:space="preserve" uid="M:System.Windows.Markup.XamlParseException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The destination for this serialization.</source>
          <target state="translated">Das Ziel dieser Serialisierung.</target>       </trans-unit>
        <trans-unit id="145" translate="yes" xml:space="preserve" uid="M:System.Windows.Markup.XamlParseException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Gets the data that is required to serialize the specified object by populating the specified <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> object.</source>
          <target state="translated">Ruft die zum Serialisieren des angegebenen Objekts durch Auffüllen des angegebenen <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph>-Objekts erforderlichen Daten ab.</target>       </trans-unit>
        <trans-unit id="146" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Windows.Markup.XamlParseException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source><ph id="ph1">&lt;xref:System.Windows.Markup.XamlParseException.GetObjectData%2A&gt;</ph> sets a <ph id="ph2">&lt;xref:System.Runtime.Serialization.SerializationInfo&gt;</ph> object with all the exception object data targeted for serialization.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Windows.Markup.XamlParseException.GetObjectData%2A&gt;</ph> Legt eine <ph id="ph2">&lt;xref:System.Runtime.Serialization.SerializationInfo&gt;</ph> Objekt mit allen Daten des Ausnahmeobjekts für die Serialisierung vorgesehen.</target>       </trans-unit>
        <trans-unit id="147" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Windows.Markup.XamlParseException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>During deserialization, the exception is reconstituted from the <ph id="ph1">&lt;xref:System.Runtime.Serialization.SerializationInfo&gt;</ph> transmitted over the stream.</source>
          <target state="translated">Während der Deserialisierung wird die Ausnahme aus der über den Stream übertragenen <ph id="ph1">&lt;xref:System.Runtime.Serialization.SerializationInfo&gt;</ph> wiederhergestellt.</target>       </trans-unit>
        <trans-unit id="148" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Windows.Markup.XamlParseException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>for providing serialization services.</source>
          <target state="translated">für das Bereitstellen von Serialisierungsdiensten.</target>       </trans-unit>
        <trans-unit id="149" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Windows.Markup.XamlParseException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Associated enumeration: <ph id="ph1">&lt;see cref="F:System.Security.Permissions.SecurityPermissionFlag.SerializationFormatter" /&gt;</ph> Security action: <ph id="ph2">&lt;see cref="M:System.Security.PermissionSet.Demand" /&gt;</ph>.</source>
          <target state="translated">Zugeordnete Enumeration: <ph id="ph1">&lt;see cref="F:System.Security.Permissions.SecurityPermissionFlag.SerializationFormatter" /&gt;</ph> Sicherheitsaktion: <ph id="ph2">&lt;see cref="M:System.Security.PermissionSet.Demand" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="150" translate="yes" xml:space="preserve" uid="P:System.Windows.Markup.XamlParseException.KeyContext">
          <source>Gets or sets the key value of the item in a dictionary where the exception occurred.</source>
          <target state="translated">Ruft den -Schlüsselwert des Elements in einem Wörterbuch fest, in dem die Ausnahme aufgetreten ist, oder legt diesen Wert fest.</target>       </trans-unit>
        <trans-unit id="151" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Markup.XamlParseException.KeyContext">
          <source>The relevant XAML <ph id="ph1">&lt;see langword="x:Key" /&gt;</ph> value.</source>
          <target state="translated">Der relevante <ph id="ph1">&lt;see langword="x:Key" /&gt;</ph>-Wert für XAML.</target>       </trans-unit>
        <trans-unit id="152" translate="yes" xml:space="preserve" uid="P:System.Windows.Markup.XamlParseException.LineNumber">
          <source>Gets the line number where the exception occurred.</source>
          <target state="translated">Ruft die Nummer der Zeile ab, in der die Ausnahme aufgetreten ist.</target>       </trans-unit>
        <trans-unit id="153" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Markup.XamlParseException.LineNumber">
          <source>The line number.</source>
          <target state="translated">Die Zeilennummer.</target>       </trans-unit>
        <trans-unit id="154" translate="yes" xml:space="preserve" uid="P:System.Windows.Markup.XamlParseException.LinePosition">
          <source>Gets the position in the line where the exception occurred.</source>
          <target state="translated">Ruft die Position in der Zeile ab, bei der die Ausnahme aufgetreten ist.</target>       </trans-unit>
        <trans-unit id="155" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Markup.XamlParseException.LinePosition">
          <source>The line position.</source>
          <target state="translated">Die Zeilenposition.</target>       </trans-unit>
        <trans-unit id="156" translate="yes" xml:space="preserve" uid="P:System.Windows.Markup.XamlParseException.NameContext">
          <source>Gets or sets the XAML name of the object where the exception occurred.</source>
          <target state="translated">Ruft den XAML-Namen des Objekts ab, in dem die Ausnahme aufgetreten ist, oder legt diesen Namen fest.</target>       </trans-unit>
        <trans-unit id="157" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Markup.XamlParseException.NameContext">
          <source>The XAML name of the object.</source>
          <target state="translated">Der XAML-Name des Objekts.</target>       </trans-unit>
        <trans-unit id="158" translate="yes" xml:space="preserve" uid="P:System.Windows.Markup.XamlParseException.UidContext">
          <source>Gets or sets the <bpt id="p1">[</bpt>x:Uid Directive<ept id="p1">](~/docs/framework/xaml-services/x-uid-directive.md)</ept> of the object where the exception occurred.</source>
          <target state="translated">Ruft die <bpt id="p1">[</bpt>x:Uid Directive<ept id="p1">](~/docs/framework/xaml-services/x-uid-directive.md)</ept>de Objekts ab, in dem die Ausnahme aufgetreten ist, oder legt sie fest.</target>       </trans-unit>
        <trans-unit id="159" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Markup.XamlParseException.UidContext">
          <source>The value of the <ph id="ph1">&lt;see langword="Uid" /&gt;</ph> string.</source>
          <target state="translated">Der Wert der <ph id="ph1">&lt;see langword="Uid" /&gt;</ph>-Zeichenfolge.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>